README.md

```diff
@@ -207,19 +207,11 @@ include the `Platform` and `VR` subdirectories when you import.
 
 1.  If you see a dialog about upgrading the Unity Oculus plugin, click
     **Accept**.
-1.  In the **Standalone** tab of the Player settings, go to **Other Settings**
-    and define `OCULUS_SUPPORTED` in **Scripting Define Symbols**. Add it to the
-    end of the list and use a semicolon to separate it from the previous define
-    commands.
```

Comment:
We haven't enabled native desktop oculus support yet! Please could you revert this?

Reply:
oh, sure, sorry!